Content.Server/_DV/CosmicCult/CosmicCultRuleSystem.cs

```diff
@@ -0,0 +1,709 @@
+using Content.Server._DV.CosmicCult.Components;
+using Content.Server._DV.CosmicCult.EntitySystems;
+using Content.Server.Actions;
+using Content.Server.Administration.Logs;
+using Content.Server.Antag;
+using Content.Server.Atmos.Components;
+using Content.Server.Audio;
+using Content.Server.Bible.Components;
+using Content.Server.Chat.Systems;
+using Content.Server.CrewManifest;
+using Content.Server.EUI;
+using Content.Server.GameTicking.Events;
+using Content.Server.GameTicking.Rules;
+using Content.Server.GameTicking;
+using Content.Server.Ghost;
+using Content.Server.Light.Components;
+using Content.Server.Objectives.Components;
+using Content.Server.Popups;
+using Content.Server.Radio.Components;
+using Content.Server.Roles;
+using Content.Server.RoundEnd;
+using Content.Server.Shuttles.Systems;
+using Content.Server.Voting.Managers;
+using Content.Server.Voting;
+using Content.Shared._DV.CCVars;
+using Content.Shared._DV.CosmicCult.Components.Examine;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Prototypes;
+using Content.Shared._DV.CosmicCult;
+using Content.Shared.Alert;
+using Content.Shared.Audio;
+using Content.Shared.Body.Systems;
+using Content.Shared.Coordinates;
+using Content.Shared.Damage;
+using Content.Shared.Database;
+using Content.Shared.GameTicking.Components;
+using Content.Shared.Humanoid;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Mind.Components;
+using Content.Shared.Mind;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs;
+using Content.Shared.Movement.Systems;
+using Content.Shared.Parallax;
+using Content.Shared.Popups;
+using Content.Shared.Roles;
+using Content.Shared.Stunnable;
+using Content.Shared.Temperature.Components;
+using Content.Shared.UserInterface;
+using Robust.Server.Audio;
```

Comment:
Remove unused using directive
```suggestion
using Content.Shared.Temperature.Components;
using Robust.Server.Audio;
```